packages/query/src/fragments/seo/template/twitter-metadata.graphql

```diff
@@ -0,0 +1,22 @@
+fragment TwitterMetadataFrag on RenderedTemplate {
+	connectedNode {
+		__typename
+		uri
+
+		... on NodeWithTitle {
+			title
+		}
+
+		... on NodeWithFeaturedImage {
+			featuredImage {
+				node {
+					sourceUrl
```

Comment:
```suggestion
				node {
					id
					sourceUrl
```

Reply:
Incorrect. ID is what allows us us to stich multiple queries together into the same cache. The fact that Apollo does this automatically doesn't change the fact that as I've said before _all nodes_ should include their global `id`>

Reply:
Before using fetch-policy: no-cache, there apollo was throwing warnings for this (missing id). Since there were no warnings after that, I was under the impression that ID is not needed anymore.